src/DDoSX/Entities/CdnRule.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace UKFast\SDK\DDoSX\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $cacheControl
+ * @property string $type
+ * @property string $uri
+ * @property array $mimeTypes
```

Comment:
nit: I've also seen `string[]` used

Reply:
I think that it should stay as an array as that is how the data is output

```
"mimeTypes" => array:1 [
        0 => "application/pdf"
      ]

```